aws/scenarios/dockerVM/main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import (
+	"github.com/DataDog/test-infra-definitions/aws/scenarios/dockerVM/dockerVM"
+
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
```

Comment:
I can see how confusing it could be to have main both here and in `datadog-agent`. we do want to define custom environments in `datadog-agent`. Do we want to disable generic environments spin up from `test-infra-definitions` ?

For example, if a team needs a basic ec2 instance, should they define a `main` in `datadog-agent` ? It can lead to duplication. The alternative is having a generic main here.

From my use case I will not use generic environments as standalone, but I can see it to be useful for some other folks.

Reply:
I don't think we want generic creations in `datadog-agent` so we should probably keep a main